tests/notification.c

```diff
@@ -243,6 +243,31 @@ test_bytes_icon ()
   test_icon (serialized_icon_s, NULL, FALSE);
 }
 
+static void
+test_file_icon ()
+{
+  g_autofree char *uri = NULL;
+  g_autofree char *serialized_icon_s = NULL;
+  g_autofree char *exp_serialized_icon_s = NULL;
+  g_autoptr(GVariant) serialized_icon = NULL;
+  g_autoptr(GFile) file = NULL;
+  g_autoptr(GIcon) icon = NULL;
+  g_autoptr(GFileIOStream) iostream = NULL;
+  GOutputStream *stream = NULL;
```

Comment:
I think this one needs `g_autoptr` as well, no?

Reply:
No, the returned output stream shouldn't be freed See: https://docs.gtk.org/gio/method.IOStream.get_output_stream.html